spec/features/can_delete_donations_from_cart_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+feature "Delete donations from cart" do
+  scenario "as guest can delete donations from two different candidates in cart" do
+    candidate = Candidate.create(name: "Ted Cruz", party: "Republican", bio: "Kim Davis")
+    issue = Issue.create(topic: "Gun Control", description: "Guns Guns Guns!")
+    candidate_issue = CandidateIssue.create(candidate: candidate, issue: issue, stance: "Give them the guns!")
+
+    candidate2 = Candidate.create(name: "Ted Cruz", party: "Republican", bio: "Kim Davis")
```

Comment:
Useless assignment to variable - `candidate2`. Did you mean `candidate`?<br>Line is too long. [90/80]